Source/ParameterEncoding.swift

```diff
@@ -96,6 +98,7 @@ public struct URLEncoding: ParameterEncoding {
                 return key
             case .indexInBrackets:
                 return "\(key)[\(index)]"
+            case let .custom(encoding): return encoding(key, index)
```

Comment:
```suggestion
            case let .custom(encoding): 
                return encoding(key, index)
```
Follow the existing indentation.